src/frontend/src/handler/create_table.rs

```diff
@@ -1094,7 +1094,9 @@ async fn derive_schema_for_cdc_table(
         let config = ExternalTableConfig::try_from_btreemap(connect_properties)
             .context("failed to extract external table config")?;
 
-        let table = ExternalTableImpl::connect(config).await?;
+        let table = ExternalTableImpl::connect(config)
+            .await
+            .context("frontend: failed to auto derive table schema")?;
```

Comment:
```suggestion
            .context("failed to auto derive table schema")?;
```

nit: I would suggest not mentioning the implementation/architecture detail in the message. We can always grep the error message in the codebase or check the backtrace to find such information.

Reply:
sounds ok to me